src/SFA.DAS.FAA.Application/Commands/Vacancy/DeleteSavedVacancy/DeleteSavedVacancyCommandHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using MediatR;
+using SFA.DAS.FAA.Domain.Interfaces;
+using SFA.DAS.FAA.Domain.SavedVacancies;
+
+namespace SFA.DAS.FAA.Application.Commands.Vacancy.DeleteSavedVacancy
+{
+    public record DeleteSavedVacancyCommandHandler : IRequestHandler<DeleteSavedVacancyCommand, Unit>
+    {
+        private readonly IApiClient _apiClient;
+
+        public DeleteSavedVacancyCommandHandler(IApiClient apiClient) => _apiClient = apiClient;
+
+        public async Task<Unit> Handle(DeleteSavedVacancyCommand request, CancellationToken cancellationToken)
+        {
+            var apiRequest = new PostDeleteSavedVacancyApiRequest(request.CandidateId, new PostDeleteSavedVacancyApiRequestData
+            {
+                VacancyReference = request.VacancyReference.Replace("VAC", string.Empty, StringComparison.CurrentCultureIgnoreCase)
```

Comment:
I understand the why 😁 but i think this should be in APIM instead and possibly not done. When we come to saving vacancies from other sources then it might become invalid

Reply:
We are already doing another check in the APIM, to keep it consistent and as recommended, I have removed it from web.